plutus-wallet-api/ledger/Ledger/Scripts.hs

```diff
@@ -250,6 +250,9 @@ newtype DataScriptHash =
     deriving stock (Generic)
     deriving newtype (Haskell.Eq, Haskell.Ord, Eq, Ord, Hashable, IsData)
 
+instance Pretty DataScriptHash where
+    pretty _ = "< hash >"
+
```

Comment:
I'd appreciate a comment explaining this definition.

Reply:
In fact, this just needs a proper pretty-printer.